hw/ip/csrng/data/csrng_testplan.hjson

```diff
@@ -183,5 +175,29 @@
             - hw0_genbits_pop_cross, hw1_genbits_pop_cross, sw_genbits_pop_cross : genbits FIFO fill status x genbits FIFO read valid x genbits FIFO read ready
             '''
     }
+    {
+      name: csrng_alert_cg
+      desc: '''
+            Covers Fatal alert and recoverable alerts
+            '''
+    }
+    {
+      name: csrng_otp_en_sw_app_read_cg
+      desc: '''
+            Covers otp_en_csrng_sw_app_read feature
+            - read_int_state_val_cp : Covers read of INT_STATE_VAL register
+            - read_genbits_reg_cp : Covers for read of GENBITS register
+            - cp_sw_app_read : Coers for values supported by otp_en_csrng_sw_app_read
```

Comment:
```suggestion
            - cp_sw_app_read : Covers for values supported by otp_en_csrng_sw_app_read
```

Reply:
Fixed typo in [8cb28ae](https://github.com/lowRISC/opentitan/compare/b75480112ac0bbf7313fda83589a19f3f96280c2..8cb28ae611d26d1f9c660443ec2f2cf790dedbc3)